samples/bookinfo/routing/certmanager-gateway.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: cert-manager-gateway
+  namespace: istio-system
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "*"
+---
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
Same - but it might be ok to keep it here if we don't want this to be opened by default. 

Maybe an example could be included in the actual VirtualService for bookinfo, with the host and path mapping.
But for 1.1 / future.